.gitlab/ci/bucket-upload.yml

```diff
@@ -54,17 +54,17 @@ create-instances-upload-flow:
     - if: '$BUCKET_UPLOAD == "true"'
     - if: '$FORCE_BUCKET_UPLOAD == "true"'
 
-#create-instances-mpv2-upload-flow:
-#  extends:
-#    - create-instances-mpv2
-#  variables:
-#    IFRA_ENV_TYPE: "Bucket-Upload"
-#    ARTIFACTS_FOLDER: "${ARTIFACTS_FOLDER_MPV2}"
-#  rules:
-#    - if: '$CI_COMMIT_BRANCH =~ /pull\/[0-9]+/'
-#      when: never
-#    - if: '$BUCKET_UPLOAD == "true"'
-#    - if: '$FORCE_BUCKET_UPLOAD == "true"'
+create-instances-mpv2-upload-flow:
+ extends:
+   - create-instances-mpv2
+ variables:
+   IFRA_ENV_TYPE: "Bucket-Upload"
+   ARTIFACTS_FOLDER: "${ARTIFACTS_FOLDER_MPV2}"
+ rules:
+   - if: '$CI_COMMIT_BRANCH =~ /pull\/[0-9]+/'
+     when: never
+   - if: '$BUCKET_UPLOAD == "true"'
+   - if: '$FORCE_BUCKET_UPLOAD == "true"'
 
 
```

Comment:
Let's add a commented-out placeholder for a job to test the successful installation of mpv2 packs to instances (which you can install mpv2 packs on).

mahsu kmo,
```
# TO-DO: Job to test installation of mpv2 packs - issue <link-here>
# install-mpv2-pack-on-mpv2-instances
```